ddl/sequence.go

```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/pingcap/tidb/parser/ast"
 	"github.com/pingcap/tidb/parser/model"
 	"github.com/pingcap/tidb/util/dbterror"
-	math2 "github.com/pingcap/tidb/util/math"
+	math2 "github.com/pingcap/tidb/util/mathutil"
```

Comment:
I would prefer to remove the dependency of github.com/cznic/mathutil and replace the Max/Min functions from there with github.com/pingcap/tidb/util/mathutil and then there will be no need for the alias.

Reply:
removed ```modernc.org/mathutil``` and ```github.com/cznic/mathutil```.